common/system/tasks/trigger_manager.cpp

```diff
@@ -82,6 +120,26 @@ bool triggerManager_setTriggerConfig(uint8_t triggerNum,
   return false;
 }
 
+const TriggerState triggerManager_getExpectedTriggers(uint8_t i) {
```

Comment:
Is calling triggerManager_checkExpectedTriggers a precondition for this function? If so, comment it?

Reply:
It gets called in `update` so it doesn't need to be explicitly called before this